apps/frontend/src/validation/criteria/ICriteriaErrors.ts

```diff
@@ -0,0 +1,3 @@
+export interface ICriteriaErrors {
+    nameErrorMsg: string
+}
```

Comment:
As mentioned in my general comment, would be nice to make this functionality generic.

Reply:
Agree, i just had to get an approval on the core functionalities before i start refactoring